client/components/data/domain-management/index.jsx

```diff
@@ -75,7 +76,8 @@ const DomainManagementData = React.createClass( {
 					context={ this.props.context }
 				/>
 				{ this.props.selectedSite &&
-					<QuerySitePlans siteId={ this.props.selectedSite.ID } />
+					<QuerySitePlans siteId={ this.props.selectedSite.ID } /> &&
+					<QueryContactDetailsCache />
```

Comment:
I'm kind of amazed that this works. I thought components were mounted when they get added to the DOM, and I would have expected that not to happen for `QuerySitePlans` because the whole expression returns `QueryContactDetailsCache`.

It does get mounted though, it doesn't need to actually render so it works! TIL.